cloudevents/working-drafts/xml-format.md

```diff
@@ -173,6 +182,13 @@ Example _(XML preamble and namespace definitions omitted for brevity)_:
 </event>
 ```
 
+The `event` element MUST NOT contain any direct child text nodes with non-whitespace
+content, and it MUST NOT contain any XML attributes other than `specversion` and attributes
+in the `xmlns` namespace.
+
+The `event` element MAY contain child elements which are not in the namespace of
+`http://cloudevents.io/xmlformat/V1`. Any such elements MUST be ignored during processing.
```

Comment:
I don't think we want the MUST there, otherwise we're saying "yes you can have extensions but you can't actually use them". I think we want something more like "Unknown extension elements MUST NOT result in an error during receive processing" or something like that.

Reply:
The original statement was aimed to SDK writers, I don't believe there's a way we can accommodate "random stuff" from a predictable processing perspective.

By that what I meant was..  _"sure, go-ahead and sprinkle your random stuff into the XML document, but if you do that don't expect any of it to be preserved across different CE Formats and transports as it doesn't form a part of the CE information model and thus is not understood by CE aware s/w and infrastructure"_

Reply:
@JemDay yes, and that's the way I'd like to see this spec go... "sure add extensions but there's no guarantees about it". In fact, that's kind of what we already do in general.... you can add new "ce-*" attributes w/o any guarantee that it'll be understood.

Reply:
There's a difference between "allowing unknown attributes that still follow our data scheme" (we're not allowing anyone to include CloudExtension attributes with unsupported xsi types here, for example) and "we allow extra data completely outside the CloudEvent data model to be intermingled with the rest of it". Anyway, let's chat about it in the meeting later - if I'm the only one who wants to be restrictive about what's valid, I'm happy to write another version of this PR which is even more liberal in what's accepted. I *do* want to be absolutely explicit about it either way though.